webdriver/tests/bidi/browsing_context/locate_nodes/max_node_count.py

```diff
@@ -0,0 +1,32 @@
+
+import pytest
+
+
+@pytest.mark.parametrize("type,value", [
+    ("css", "div"),
+    ("xpath", "//div"),
+    ("innerText", "foo")
+])
+@pytest.mark.asyncio
+async def test_find_by_css_limit_return_count(bidi_session, inline, top_context, type, value):
+    url = inline("""<div data-class="one">foo</div><div data-class="two">foo</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    result = await bidi_session.browsing_context.locate_nodes(
+        context=top_context["context"],
+        locator={ "type": type, "value": value },
+        max_node_count = 1
```

Comment:
We probably should parametrize here as well and use `1` as well as `10` (which only returns two nodes).

Reply:
Done.